### ISSUE TITLE
cursor: false option not working

### ISSUE THREAD

**Issue by a-ssassi-n** [alexmacarthur/typeit#43]
My code totally works fine, but when I apply the `cursor: false` option, it doesn't seems to work. (Even the sting disappears)

```
 $('#typename').typeIt({
        strings: 'HelloWorld',
        speed: 100,
        autoStart: false,
        loop: false,
       //cursor: false
    });
```
Using version: 4.4.0

**Comment by alexmacarthur** [comment]
Hi! Thanks for finding that bug! It looks like the cursor toggle is working -- it's an issue with the `autoStart`. If your element is inline, it won't be visible by the plugin and so it won't trigger. 

Until a bug fix is released, you can address this by setting a minimum width on your element, so it's visible in the DOM:

https://codepen.io/alexmacarthur/pen/jLXvWd

**Comment by alexmacarthur** [comment]
Resolved in version 4.4.1.
